Unit tests fail during Maven build

When doing a `mvn test`, the build fails with several error messages. There seems to be a dependency missing: 

`[...] Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory`

When skipping tests, the build works.

--- a/spring-boot/admin/pom.xml
+++ b/spring-boot/admin/pom.xml
@@ -254,6 +254,12 @@
             <version>2.9.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.3</version>
+            <scope>runtime</scope>
+        </dependency>
 
     </dependencies>
 

